src/screens/LoginScreen.tsx

```diff
@@ -316,6 +317,7 @@ async function loginWithID(participantID: string, dispatch) {
       trialLength: experimentApiData.experiment.trial_length * 1000,
       ratingDelay: experimentApiData.experiment.rating_delay * 1000,
       modules,
+      minimumVolume: experimentApiData.experiment.minimum_volume,
```

Comment:
**Question:** Should this have a fallback value or are we always expecting a value to be set in `experimentRawData`'s fetch request

Reply:
Yes it will always be set, The database has a default value of 1.

Reply:
Nice, just noticed that from doing the BE review